pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -371,6 +372,8 @@ private static void start() throws MalformedURLException {
         post(API.ADMIN_STUDY_USER_LOGIN_ACCOUNT, new AdminCreateUserLoginAccountRoute(), jsonSerializer);
         post(API.ADMIN_STUDY_PARTICIPANTS_LOOKUP,
                 new ParticipantsLookupRoute(new ESParticipantsLookupService(esClient)), responseSerializer);
+        get(API.ADMIN_STUDY_PARTICIPANT_LOOKUP_BY_GUID,
+                new ParticipantLookupByGuidRoute(new ESParticipantsLookupService(esClient)), responseSerializer);
```

Comment:
Can we rename the class to `AdminParticipantLookupByGuidRoute`? Let's add `Admin` prefix to `ParticipantsLookupRoute` too.

Reply:
Fixed